swift/ql/lib/codeql/swift/frameworks/StandardLibrary/IteratorProtocol.qll

```diff
@@ -0,0 +1,17 @@
+/**
+ * Provides models for the `IteratorProtocol` Swift protocol.
+ */
+
+private import codeql.swift.dataflow.ExternalFlow
+
+/**
+ * A model for `IteratorProtocol` members that permit taint flow.
+ */
+private class IteratorProtocolSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        ";IteratorProtocol;true;next();;;Argument[-1].CollectionElement;ReturnValue.OptionalSome;value"
```

Comment:
Now that we have a model for `IteratorProtocol.next()`, do you think there are any other iterator-related things we should plan to model in the near future?

Reply:
I don't think so - IteratorProtocol is just `next()` and provides nothing extra.